src/tutorial/runtime/introduction.rst

```diff
@@ -385,11 +384,11 @@ Numbered run directories
 
 .. code-block:: console
 
-   $ cylc validate my_workflow
+   $ cylc play my_workflow
```

Comment:
Fine but what's the reason?

Reply:
Earlier on L309 we say run `cylc validate ~/cylc-src/my_workflow`, so I wanted to avoid any implication that `cylc validate my_workflow` is short for that